### ISSUE TITLE
One question for the CIFAR-100 experiment

### ISSUE THREAD

**Issue by yuhodots** [CanPeng123/FSCIL_ALICE#3]
Hi. Thank you for releasing the code!
I have one question for the CIFAR-100 experiment.

All existing FSCIL methods used ResNet-20(0.27M params) for CIFAR-100, not ResNet-18(11 M params).
However, in this paper, it seems that ResNet-18 was used in the CIFAR-100 experiment, so I wonder why you used a different backbone architecture.

Thanks.

**Comment by yuhodots** [comment]
Additionally, I wonder if there was any special reason to use Pre-activation ResNet rather than ResNet. 
(for CIFAR-100 and miniImageNet)

**Comment by CanPeng123** [comment]
Hi,

People prefer to use resnet20 for CIFAR because it normally provides a better accuracy result than resnet18 since CIFAR has a smaller image size than ImageNet. In this paper, we do not find resnet18 adversely affects the CIFAR performance so we keep the backbone network as resnet18 to be consistent for all three datasets. If you are interested, you could try to replace the backbone network with resnet20 but you might need to retune the hyperparameters.

**Comment by yuhodots** [comment]
Thank you for your reply. 
Although ResNet-18 has a large number of parameters, I agree that ResNet-20 (with conventional cross-entropy loss) generally shows better performance compared to ResNet-18 on CIFAR-100.
I am curious about ALICE's performance for ResNet-20, but I will check it personally.
